tests/reference/wat-print_str-385e953.stdout

```diff
@@ -1,76 +1,484 @@
+        drop
+        i32.const 176
+        i32.const 184
+        i32.store offset=0 align=1
+        i32.const 180
+        i32.const 1
+        i32.store offset=0 align=1
+        i32.const 1
+        i32.const 176
+        i32.const 1
+        i32.const 0
+        call 1
+        drop
+        i32.const 1
+        i32.const 236
+        i32.const 244
+        i32.store offset=0 align=1
+        i32.const 240
         i32.const 6
-        call 4
-        call 5
-        i32.const 28
+        i32.store offset=0 align=1
+        i32.const 236
+        i32.const 1
+        i32.const 0
+        call 1
+        drop
+        i32.const 176
+        i32.const 184
+        i32.store offset=0 align=1
+        i32.const 180
+        i32.const 1
+        i32.store offset=0 align=1
+        i32.const 1
+        i32.const 176
+        i32.const 1
+        i32.const 0
+        call 1
+        drop
+        i32.const 1
+        i32.const 252
+        i32.const 260
+        i32.store offset=0 align=1
+        i32.const 256
         i32.const 6
-        call 4
-        call 5
+        i32.store offset=0 align=1
+        i32.const 252
+        i32.const 1
+        i32.const 0
+        call 1
+        drop
+        i32.const 176
+        i32.const 184
+        i32.store offset=0 align=1
+        i32.const 180
+        i32.const 1
+        i32.store offset=0 align=1
+        i32.const 1
+        i32.const 176
+        i32.const 1
+        i32.const 0
+        call 1
+        drop
         return
     )
-    (func $10 (type 10) (param) (result)
+    (func $7 (type 7) (param) (result)
         (local)
-        call 7
+        call 4
         i32.const 0
-        call 6
+        call 0
         return
     )
-    (export "_lpython_main_program" (func $7))
-    (export "main0" (func $8))
-    (export "printCompOperations" (func $9))
-    (export "_lcompilers_main" (func $10))
-    (data (;0;) (i32.const 0) "x < y")
-    (data (;1;) (i32.const 5) "x !< y")
-    (data (;2;) (i32.const 11) "x > y")
-    (data (;3;) (i32.const 16) "x !> y")
-    (data (;4;) (i32.const 22) "x == y")
-    (data (;5;) (i32.const 28) "x != y")
+    (memory (;0;) 100 100))
+    (export "memory" (memory 0))
+    (export "print_i64" (func 2))
+    (export "print_f64" (func 3))
+    (export "_lpython_main_program" (func 4))
+    (export "main0" (func 5))
+    (export "printCompOperations" (func 6))
+    (export "_start" (func 7))
+    (data (;0;) (i32.const 12) "-")
+    (data (;1;) (i32.const 24) "#")
+    (data (;2;) (i32.const 36) "0")
+    (data (;3;) (i32.const 48) "1")
+    (data (;4;) (i32.const 60) "2")
+    (data (;5;) (i32.const 72) "3")
+    (data (;6;) (i32.const 84) "4")
+    (data (;7;) (i32.const 96) "5")
+    (data (;8;) (i32.const 108) "6")
+    (data (;9;) (i32.const 120) "7")
+    (data (;10;) (i32.const 132) "8")
+    (data (;11;) (i32.const 144) "9")
+    (data (;12;) (i32.const 156) ".")
+    (data (;13;) (i32.const 168) "x < y")
+    (data (;14;) (i32.const 184) "
+")
```

Comment:
`newline` character (which is needed/printed by `print` after printing all arguments) is being shown as above in the generated  `WAT`.

Reply:
This has been fixed. The `newline` character is now **escaped** and is printed in its `byte` form (`\0a`) as follows:

```
(data (;3;) (i32.const 24) "\0a   ")
```